src/schema/artwork/index.js

```diff
@@ -29,6 +29,9 @@ import {
   GraphQLList,
   GraphQLInt,
 } from "graphql"
+import AttributionClass from "schema/artwork/attribution_class"
+// Mapping of attribution_class ids to AttributionClass values
+import attributionClasses from "../../lib/attribution_classes.js"
```

Comment:
Not sure what the right pattern for filenames is here — I know there's [active talk about 🐍 vs 🐫 right now](https://artsy.slack.com/archives/C1HH3KNJG/p1519931489000527).

Reply:
Oh... I did not even realize that there are both versions here in the project:( Just looked at the one above and copied the convention....

Reply:
Though reading the discussion you linked above those file should be camelCased, no? Did I interpret it correctly?

Reply:
Yea 👍 on 🐫 !!